noisepagecontrol/control_plane/services/resource_manager/initialise_resource.py

```diff
@@ -0,0 +1,32 @@
+import os
+import logging
+
+from django.conf import settings
+
+logger = logging.getLogger("control_plane")
+
+
+def initialise_resource(tuning_id, resource_type):
+    """
+    Creates a resource entry which tracks the workload
+    to be collected. This entry would be updated
+    in the future when the workload arrives
+
+    Returns: resource_id
+    Unique identifier for the inited resource
+    """
+
+    from .models import Resource
+
+    resource = Resource(
+        tuning_id=tuning_id,
+        resource_type=resource_type,
+        available=False,
+    )
+    resource.save()
+
+    # Make new dir for resource
```

Comment:
We prolly don't have to mkdir here and can just mkdir in save_resource.py when the dir does not exist yet? Right now we have duplicated `resource_dir = settings.RESOURCE_DIR / resource.resource_id` here and in save_resource.py, so it's easier to forget to modify both when we have to change resource_dir path (like changing to RESROUCE/dir/foo/resource_id)